src/builderTool.js

```diff
@@ -103,7 +103,7 @@ module.exports.buildForm = (inputs, action = false, method = 'GET') => {
 
     //this checks to see if a method was set
     if(method){
-        outputString += ` method="${method}">`;
+        outputString += ` method="${method}">\n`;
     }
 
     // Check if inputs variable is an array or throw exception
```

Comment:
Just adding some breaks in each of the elements lines so that the output in the HTML is easier to read.